app/models/submission.rb

```diff
@@ -29,8 +29,9 @@ class Submission < ActiveRecord::Base
     },
   }
 
-  ZIP_CODES = ZipBoundary.pluck(:name)
-  CENSUS_CODES = CensusBoundary.pluck(:geo_id)
+  #TODO we can't keep storing these, we need to replace all usage with a endpoint for searching boundaries
```

Comment:
This is an interesting storage problem.

We're currently searching an increasing database table for the boundary_id for all zip codes?

Then is the boundary_id is cross-referenced with another database table that gives you a polygon area?

Reply:
On startup of the Ruby service, we are getting list ZIP and Census Tracts. Prior to this PR we were using one of the two lists when getting map data, data explorer results, and the dropdowns on the UI. This PR removes two of the cases switching them joining submission data to enabled boundaries but doesn't touch the dropdown size.

Resolving the dropdown issue will require not sending all the boundary ids and instead requiring users to search, which gets search results via a new endpoint. Want to work on that?